packages/message-box/message-box.scss

```diff
@@ -34,20 +31,40 @@ $message-width: rem(560px);
         max-width: 100%;
     }
 
-    &--info:before {
-        background-color: $info;
+    &--info {
+        border: 1px solid $info;
+        background-color: change-color($info, $lightness: 97%);
```

Comment:
Mulig jeg ikke forstår denne funksjonen rett, men åpner vi ikke for å lage mange nyanser av fargene nå? Burde ikke disse komme fra et felles sted, så ikke nestemann velger lightness 94%?

Reply:
Dette er en innebygget funksjon i sass 😊 Det er nok mulig at disse fargene bør bli variabler heller enn å defineres direkte som her, men tanken er at de skal være en funksjon av "hoved"-meldingsfargen.

Reply:
Ser den, men hvis vi strør rundt oss i koden med forskjellige verdier at lightness, så blir det det en variant av kaos som vil gjerne vil unngå med et design system?

Reply:
Utöker med färgvariabler, mr @lfbergee  har en bra poäng